docs/src/IconButton.doc.js

```diff
@@ -12,202 +12,357 @@ const card = c => cards.push(c);
+          'Padding options are 1-5 representing the padding in boints.',
+        ],
+        href: 'padding',
+      },
+      {
+        name: 'rel',
+        type: `'none' | 'nofollow'`,
+        required: false,
+        defaultValue: 'none',
+        description: 'Optional with link-role buttons.',
+        href: 'roles',
       },
       {
         name: 'ref',
-        type: "React.Ref<'button'>",
-        description: 'Forward the ref to the underlying button element',
-        href: 'refExample',
+        type: `React.Ref<'button'> | React.Ref<'a'>`,
+        description:
+          'Forward the ref to the underlying button or anchor element',
+        href: 'ref',
+      },
+      {
+        name: 'role',
+        type: `'button' | 'link'`,
+        required: false,
+        defaultValue: 'button',
+        description: [
+          `Select a button variant:`,
+          `- 'button': Use to render 'submit' or 'button'-type buttons. The button is rendered as a '<button>'.`,
+          `- 'link': Use for buttons to act like links. The button is rendered as an '<a>'.`,
+          `Required with link-role buttons.`,
+        ],
+        href: 'roles',
       },
       {
         name: 'selected',
         type: 'boolean',
+        required: false,
         defaultValue: false,
-        href: 'selectedCombinations',
+        description: [
+          'Control the "selected" state of a button component to toggle binary states.',
+          'Optional with button-role + button-type buttons.',
+        ],
+        href: 'selected',
       },
       {
         name: 'size',
         type: `"xs" | "sm" | "md" | "lg" | "xl"`,
-        description: `xs: 24px, sm: 32px, md: 40px, lg: 48px, xl: 56px`,
+        required: false,
         defaultValue: 'md',
-        href: 'sizeCombinations',
+        description:
+          'Display an IconButton in different sizes. Size changes the component padding modifying its fixed height: xs (24px), sm (32px), md (40px), lg (48px), xl (56px).',
+        href: 'size',
+      },
+      {
+        name: 'target',
+        type: `null | 'self' | 'blank'`,
+        required: false,
+        defaultValue: 'null',
+        description: [
+          'Define the frame or window to open the anchor defined on `href`:',
+          `- 'null' opens the anchor in the same window.`,
+          `- 'blank' opens the anchor in a new window.`,
+          `- 'self' opens an anchor in the same frame.`,
+          'Optional with link-role buttons.',
+        ],
+        href: 'roles',
       },
     ]}
   />
 );
 
 card(
   <Example
-    name="Example"
+    name="Basic IconButton"
+    id="basic"
     defaultCode={`
 <IconButton
-  accessibilityLabel="Love"
-  bgColor="white"
-  icon="heart"
-  iconColor="red"
-  onClick={() => { console.log('❤️')}}
+  accessibilityLabel="Open edit modal to edit Board"
+  bgColor="gray"
```

Comment:
@AlbertCarreras can we remove the `bgColor="gray"`? We don't want most buttons to have a bgColor

Reply:
DONE